configure.ac

```diff
@@ -216,6 +216,39 @@ if test x"$lightgrep" == x"yes"; then
   LDFLAGS="$LDFLAGS `$PKG_CONFIG --libs-only-L --libs-only-other lightgrep`"
 fi
 
+################################################################
+## yara-x support
+##
+AC_ARG_ENABLE([yara],
+              AS_HELP_STRING([--disable-yara], [Disable YARA scanning]),
+              [],
+              [AC_DEFINE(USE_YARA, 1, [Use YARA scanning]) yara="yes"])
+if test x"$yara" == x"yes"; then
+  m4_ifndef([PKG_CHECK_MODULES],
+            [AC_MSG_ERROR([pkg-config autoconf macros are missing; try installing pkgconfig])])
+
+  if test x"$mingw" == x"yes" ; then
+    # get static flags when cross-compiling with mingw
+    PKG_CONFIG="$PKG_CONFIG --static"
+  else
+    # pkg-config doesn't look in /usr/local/lib on some systems
+    if test x"$PKG_CONFIG_PATH" != x; then
+      export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/usr/local/lib/pkgconfig
+    else
+      export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
+    fi
+  fi
```

Comment:
I want to consolidate this pkg-config block with the one above it for lightgrep to avoid duplication.